test_unstructured_ingest/expected-structured-output/local-single-file-with-pdf-infer-table-structure/layout-parser-paper.pdf.json

```diff
@@ -300,8 +290,8 @@
     "text": "Recent years have also seen numerous eﬀorts to create libraries for promoting reproducibility and reusability in the ﬁeld of DL. Libraries like Dectectron2 [35],"
   },
   {
-    "type": "ListItem",
-    "element_id": "bbde5bc98ffe50bc4557c848cb1a0473",
+    "type": "NarrativeText",
```

Comment:
the new model seem to miss classify this footer style list item as narrative text often cc @benjats07 area to fine tune